polars/polars-lazy/polars-plan/src/dsl/functions.rs

```diff
@@ -735,6 +738,8 @@ macro_rules! prepare_binary_function {
 }
 
 /// Apply a closure on the two columns that are evaluated from `Expr` a and `Expr` b.
+///
+/// The closure takes two arguments, each a `Series`. `output_type` must be the element type of the resulting `Series`.
```

Comment:
~The closure takes two arguments, each a `Series`. `output_type` must be the element type of the resulting `Series`.~

he closure takes two arguments, each a `Series`. `output_type` must be the output dtype of the resulting `Series`.